.github/workflows/build_prs_test_report_postgres10.yml

```diff
@@ -0,0 +1,47 @@
+name: PR -> PostgreSQL 10
+
+on:
+  pull_request:
+    paths-ignore:
+      - frontend/**
+      - website/**
+      - coverage/**
+
+jobs:
+  testreport-postgres10:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres:10 # Specify PostgreSQL version 10
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres10
```

Comment:
Can we use the matrix?

Reply:
Do you mean something like this? https://github.com/Unleash/unleash/pull/5385/commits/ab83c7b118ed8f6d063753f125f3561ceef49c65

Reply:
Actually, I thought about using the existing workflow